python/fastsim/__init__.py

```diff
@@ -33,6 +33,39 @@ def package_root() -> Path:
 
 __version__ = get_distribution("fastsim").version
 
+#download demo files
```

Comment:
@calbaker this is what I have so far, but I'm somewhat stumped, any ideas?

Reply:
@robinsteuteville , is this thread resolved?  If yes, mark the 
<img width="198" alt="image" src="https://github.com/NREL/fastsim/assets/854536/2d2d637e-2eb7-4a35-a830-91873953c2bd">
button below.